db/migrate/20180307074200_rename_organization_to_team.rb

```diff
@@ -0,0 +1,6 @@
+class RenameOrganizationToTeam < ActiveRecord::Migration[5.0]
+  def change
+    rename_table :organizations, :teams
+    rename_column :roles, :organization_id, :team_id
```

Comment:
misschien wel een idee om gewoon 1 keer handmatig zo'n pg export te pakken van de u-can-act production db en dan alle migraties van jouw en mijn pullrequests te draaien voordat we onze hele database op production hosen.

Reply:
id-f-d